### DISCUSSION TITLE
Example of cmdk as modal with overlay

### DISCUSSION THREAD

**Discussion by 4ortytwo** [pacocoursey/cmdk#218] [Q&A]
Hi everyone! Looking for an example of cmdk as a modal with overlay, not sure how to implement it.

**Comment by hbeus** [comment]
What part do you need help with? The styling?
There's a prop on Command.Dialog, overlayClassName, to add styles to the overlay.

**Comment by marcelo-earth** [comment]
I don't know how to do it either, the examples are not enough unfortunately :(
